openpype/hosts/maya/plugins/publish/validate_shots_script_nodes.py

```diff
@@ -0,0 +1,34 @@
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import (
+    OptionalPyblishPluginMixin,
+    ValidateContentsOrder,
+    PublishValidationError
+)
+
+
+class ValidateShotsSceneConfigurationScriptNode(
+    pyblish.api.ContextPlugin, OptionalPyblishPluginMixin
+):
+    """Ensure script nodes are correctly named for later extraction."""
```

Comment:
It might be good to add in the docstring why we want the script node to exist so that it's clear why this node is required to begin with?